.travis.yml

```diff
@@ -1,14 +1,13 @@
 language: php
 
+php:
+    - 5.6
+    - 7.0
+    - 7.1
+    - hhvm
+
 matrix:
-    include:
-        - php: 5.5
-          env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
-        - php: 5.5
-          env: 'BOX=yes'
-        - php: 5.6
-        - php: 7.0
-          env: CHECKS=yes
```

Comment:
too much changes
- 5.5 should be there, especially the flags !
- checks should be there on highest php version
- why you removed the box ?